velox/exec/tests/AssignUniqueIdTest.cpp

```diff
@@ -83,11 +83,13 @@ TEST_F(AssignUniqueIdTest, exceedRequestLimit) {
 }
 
 TEST_F(AssignUniqueIdTest, multiThread) {
-  vector_size_t batchSize = 1000;
-  auto input = {makeRowVector(
-      {makeFlatVector<int32_t>(batchSize, [](auto row) { return row; })})};
+  for (int i = 0; i < 3; i++) {
```

Comment:
Is this loop needed?

Reply:
@majetideepak I will prefer to keep this loop so the unit test has higher chance to fail, instead of flakiness